test/api.ts

```diff
@@ -187,7 +192,7 @@ suite('api v1', function () {
             @Struct.field('asset') quantity!: Asset
             @Struct.field('string') memo!: string
         }
-        const info = await jungle.v1.chain.get_info()
```

Comment:
Why is this changed?

Reply:
I think it's because I was having problems with caches early on, where it was loading the expired get_info request and then submitting a transaction that was long expired. When I made that change, I was also using the `jungle` instance for all the new tests - which is what was changing that cache.

Reply:
Reverted those tests back to using Jungle in https://github.com/greymass/eosio-core/pull/29/commits/90358793963e4fd4cf60ac14e42d8509802cb3e3